docs/additional_information/credits.rst

```diff
@@ -4,38 +4,72 @@ Contact and Credits
 ===================
 
 If you have any questions or comments, please do not hesitate to contact us via filing
-an issue on Github or directly via email.
+an issue on Github, writing an `email`_ to our `zulipchat group
+<http://gettsim.zulipchat.com/>`_ or via personal emails.
+
+.. _email: respy.9b46528f81292a712fa4855ff362f40f.show-sender@streams.zulipchat.com
+
 
 Development Lead
-^^^^^^^^^^^^^^^^
+----------------
+
+`Tobias Raabe <https://github.com/tobiasraabe>`_
 
-`Philipp Eisenhauer <https://github.com/peisenha>`_
 
-Contributors
-^^^^^^^^^^^^
+Maintainer / Core Developer
+---------------------------
+
+- `Philipp Eisenhauer <https://github.com/peisenha>`_
+- `Janos Gabler <https://github.com/janosg>`_
 
-| `Janos Gabler <https://github.com/janosg>`_
-| `Boryana Ilieva <https://github.com/boryana-ilieva>`_
-| `Tobias Raabe <https://github.com/tobiasraabe>`_
 
 Acknowledgments
-^^^^^^^^^^^^^^^
+---------------
 
 We are grateful to the `Social Science Computing Services <https://sscs.uchicago.edu/>`_
 at the `University of Chicago <https://www.uchicago.edu/>`_ who let us use the Acropolis
 cluster for scalability and performance testing. We appreciate the financial support of
 the `AXA Research Fund <https://www.axa-research.org/>`_ and the  `University of Bonn
 <https://www.uni-bonn.de>`_. We are indebted to the open source community as we build on
-top of numerous open source tools such as the `SciPy Stack <https://www.scipy.org>`_,
-and `waf <https://waf.io/>`_.
+top of numerous open source tools such as the `SciPy <https://www.scipy.org>`_ and
+`PyData <https://pydata.org/>`_ ecosystems.
```

Comment:
We should mention Numba and Pandas explicitly and not only the SciPy and PyData ecosystem. Without Numba it would be completely impossible to get respy that fast. Without pandas, the whole interface would not work.